PowerValidatedSolutions.psm1

```diff
@@ -26335,6 +26348,69 @@ Function Get-vRACloudZone {
 }
 Export-ModuleMember -Function Get-vRACloudZone
 
+Function Get-vRAAPIVersion {
+    <#
+        .SYNOPSIS
+        Retrieve the vRealize Automation version information
+        
+        .DESCRIPTION
+        The GGet-vRAAPIVersion cmdlest retrieves the ApiVersion of vRealize Automation
+
+        .EXAMPLE
+        Get-vRAAPIVersion 
+    #>
+    Try {
+        $uri = "https://$vraAppliance/iaas/api/about"
+        $response = Invoke-RestMethod -Method 'GET' -Uri $uri -Headers $vraHeaders
+        $response.latestApiVersion         
+    }
+    Catch {
+        Write-Error $_.Exception.Message
+    }
+}
+Export-ModuleMember -Function Get-vRAAPIVersion
+
+Function Get-vRAIntegrationDetail {
+    <#
+        .SYNOPSIS
+        Get an integration detail of an item from vRealize Automation
+
+        .DESCRIPTION
+        The Get-vRAIntegrationDetail cmdlet get an integration details of an item from vRealize Automation
+
+        .EXAMPLE
+        Get-vRAIntegrationDetail -integrationType "vrops" -getVCID
+        This example gets the vCenter Servers ids managed by the vRealize Operations Manager
```

Comment:
"returns the ids of the vCenter Server instances"

Reply:
Done